### ISSUE TITLE
SQL Server feature: Full Text Search

### ISSUE THREAD

**Issue by DannyMeister** [gusztavvargadr/packer#78]
Would you entertain the idea of enabling Full Text Search in SQL Server?  That feature cannot be enabled after the server is created, so I cannot use a provisioning script to turn it on.

**Comment by gusztavvargadr** [comment]
@DannyMeister thank you for the suggestion.

Do you prefer SQL 2014 or 2017 for this and #79? I have limited build resources so maintaining only one of them helps me a lot (officially my 2014 boxes are obsolete now so I'd prefer to do it only for 2017, but I see 2014 is still used heavily, so I may extend maintenance for that as well).

Technically I'd probably consider a core engine only and a full version (this is usually a tradeoff between build resources, image size and features).

**Comment by DannyMeister** [comment]
Our software is certified on sql2014 only at this point, so I'll have to stick with the 2014 box until we update our own compatibility (working on pushing for that one... sometimes big ships turn slowly), or any time i have to work on issues on a prior version.  I would understand if you only enhanced the 2017 box though.  I still get good use out of your 2014 box for any of the applications which don't require these features.

A core box and full version sounds wonderful if you have the build resources and time to keep up with both.  It's nice to use a slimmer image when possible.

**Comment by gusztavvargadr** [comment]
@DannyMeister ok, i'll see what i can do. I can try to include this in the current (about to publish) or the upcoming release.

**Comment by gusztavvargadr** [comment]
@DannyMeister if you have some time (and bandwidth :), I have some new release candidates with the full text search included (only for Hyper-V for now, VirtualBox is coming soon):

- https://app.vagrantup.com/gusztavvargadr/boxes/w16s-sql14d
- https://app.vagrantup.com/gusztavvargadr/boxes/w16s-sql17d

It would be nice to have your feedback if they work as expected, thanks!

**Comment by jagossel** [comment]
@gustavvargadr I appreciate your time ans willingness to create these boxes. Love the fact it was done via Sysprep that allows the evaluation period to be reset.

I work on the same application suite as @DannyMeister and I am using your w16s-iis box for VirtualBox. I would love to use the VirtualBox version of the w16s-sql14d or w16s-sql17d, as it would save me a lot of time from having the w16s-iis installing SQL Server 2012 Express with Advanced Services from Chocolatey.

**Comment by gusztavvargadr** [comment]
@jagossel thanks for the feedback, I'll build the VirtualBox ones next (just that I have one machine for building now and I personally use Hyper-V, so I start with those - cannot run both providers in parallel) and will get back to you soon.

**Comment by gusztavvargadr** [comment]
@jagossel, @DannyMeister release candidates are now available for VirtualBox as well:

- https://app.vagrantup.com/gusztavvargadr/boxes/w16s-sql14d
- https://app.vagrantup.com/gusztavvargadr/boxes/w16s-sql17d
